tests/src/FunctionalJavascript/ModuleIntegrationTest.php

```diff
@@ -393,4 +395,182 @@ public function testDevicePreview() {
+    $this->setFieldValue($page,
+      'field_embed_media[0][subform][field_media][0][inline_entity_form][field_url][0][uri]',
+      'https://twitter.com/tweetsauce/status/778001033142284288'
+    );
+
+    $this->liveblogSetBody($page, 'Very nice twitter post you have here!');
+
+    $this->liveblogSubmit();
+
+    $this->waitUntilVisible('article[data-postid="3"]', 10000);
+    $this->waitUntilVisible('[data-tweet-id="778001033142284288"]', 10000);
+
+    // Add post with instagram.
+    $this->liveblogSetTitle($page, 'Instagram post');
+
+    $this->clickDropButton('field_embed_media_instagram_add_more');
+    $this->setFieldValue($page,
+      'field_embed_media[0][subform][field_media][0][inline_entity_form][field_url][0][uri]',
+      'https://www.instagram.com/p/BNU5k6jhds9/'
+    );
+
+    $this->liveblogSetBody($page, 'Very nice instagram post you have here!');
+
+    $this->liveblogSubmit();
+
+    $this->waitUntilVisible('article[data-postid="4"]', 10000);
+    $this->waitUntilVisible('iframe.instagram-media-rendered', 10000);
+
+    // Check site with anonymous user.
+    $url = $this->getUrl();
+    $this->drupalLogout();
+
+    $this->drupalGet($url);
+
+    $this->waitUntilVisible('article[data-postid="4"]');
+    $this->assertSession()->elementNotExists('css', 'article[data-postid="3"]');
+    $this->assertSession()->elementNotExists('css', 'article[data-postid="2"]');
+    $this->assertSession()->elementNotExists('css', 'article[data-postid="1"]');
+
+    $this->scrollElementInView('article[data-postid="4"]');
+    $this->waitUntilVisible('article[data-postid="3"]');
+    $this->waitUntilVisible('article[data-postid="2"]');
+    $this->waitUntilVisible('article[data-postid="1"]');
+  }
+
+  /**
+   * Set the title of a liveblog post.
+   *
+   * @param \Behat\Mink\Element\DocumentElement $page
+   *   Current active page.
+   * @param string $title
+   *   The title.
+   */
+  protected function liveblogSetTitle(DocumentElement $page, $title) {
+    $this->setFieldValue($page, 'title[0][value]', $title);
+  }
+
+  /**
+   * Set the body of a liveblog post.
+   *
+   * @param \Behat\Mink\Element\DocumentElement $page
+   *   Current active page.
+   * @param string $body
+   *   The body.
+   */
+  protected function liveblogSetBody(DocumentElement $page, $body) {
+    $this->fillCkEditor(
+      $page,
+      "textarea[name='body[0][value]']",
+      $body
+    );
+  }
+
+  /**
+   * Submit the liveblog post.
+   */
+  protected function liveblogSubmit() {
+    $this->scrollElementInView('[data-drupal-selector="edit-submit"]');
+    $this->click('input[data-drupal-selector="edit-submit"]');
+  }
+
+  /**
+   * Click a button within a dropdown button field.
+   *
+   * @param string $fieldName
+   *   The [name] attribute of the button to be clicked.
+   * @param bool $toggle
+   *   Whether the dropdown button should be expanded before clicking.
+   */
+  protected function clickDropButton($fieldName, $toggle = TRUE) {
```

Comment:
It should not depend on any type of paragraph -> but I have forgot that we have switched to modal dialog for adding paragraphs. In your case you can't use that method.

So just make your function generic (if it's not already) and move to base.